### ISSUE TITLE
Flame of Azzinoth "Blaze" damage is too low

### ISSUE THREAD

**Issue by Dorakrab** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3748]
Current situation : 
On phase 2 of Illidan fight Flame of Azzinoth "Blaze" ability's damage is too low and 2 healers are able to outheal it even if the tank is not wearing any fire resist gear at all. So if you get lucky enough it is possible that tanks wouldn't need to move the adds at all. Also the adds use their blaze ability not as frequently as they should, they should cast it roughly every 10 seconds (see video below) but they cast in about every 30 seconds.

Desired situation : 
Blaze damage should not be healable and tanks should constantly move Flame of Azzinoth adds. Blaze should be casted every 10 seconds.

Wowwiki damage value : 
Blaze: AoE dealing 5k fire damage per second, centered on the ground at current target. Blaze always appears shortly after Flame Blast, and its location is decided at the instant the Flame Blast goes off. 

http://wowwiki.wikia.com/wiki/Illidan_Stormrage_%28tactics%29

Basically this is how tanking on phase 2 should look like : http://www.youtube.com/watch?v=iaYFMXLJcoQ instead of this https://youtu.be/hCQ0RZeAIeY?t=7m23s

**Comment by Dorakrab** [comment]
"I would usher you to please refresh your topic within the bug tracker so that our development team may do some further testing and work out any issues affecting this spell"
